bokehjs/src/coffee/models/layouts/layout_dom.coffee

```diff
@@ -38,9 +38,14 @@ class LayoutDOMView extends BokehView
       _.delay(resize, 100)
       _.delay(resize, 200)
 
+    if @model.responsive is 'fixed'
+      # Render isn't triggered by resize - so trigger manually
+      @render()
+
   bind_bokeh_events: () ->
     @listenTo(@model, 'change', @render)
-    @listenTo(@model.document.solver(), 'resize', @render)
+    if @model.responsive isnt 'fixed'
+      @listenTo(@model.document.solver(), 'resize', @render)
```

Comment:
If `responsive` is changed, do we get this event handler rewired to get the expected behavior?